source/project-stories/contiki-holidays.erb

```diff
@@ -81,6 +81,7 @@ team:
     - short_name: "heidi"
       full_name: "Heidi Geisler"
       link: "/people/heidi-geisler"
+  people_archived:
     - short_name: "ben-j"
       full_name: "Ben Janecke"
       link: "/people/ben-janecke"
```

Comment:
Oops, late PR sometimes == missing things :) Thanks for catching this up! Fixed.